EIPS/eip-draft_grant_standard.md

```diff
@@ -9,26 +9,27 @@ category: ERC
 created: 2019-04-30
 ---
 
-## Abstract
+## Simple Summary
 <!--"If you can't explain it simply, you don't understand it well enough." Provide a simplified and layman-accessible explanation of the EIP.-->
 This document outlines a standard interface to propose, vote on, and distribute grants.
 
+## Abstract
+This standard specifies a way for Ethereum users to propose, vote on and manage the distribution of grants. It allows for multiple grant recipients to be listed and receive rewards through one smart contract. The author of the contract may specify certain conditions such as requiring atomic grants (i.e. raise all or none), a voting threshold with Ether or other tokens, and a list of grant managers that can release funds over time.
+
+
 ## Motivation
 <!--The motivation is critical for EIPs that want to change the Ethereum protocol. It should clearly explain why the existing protocol specification is inadequate to address the problem that the EIP solves. EIP submissions without sufficient motivation may be rejected outright.-->
 
-In short, the motivation is social scalability. We believe that a grant standard is necessary to coordinate grant efforts across the Ethereum community. Having a specified interface will more easily enable wallets, DAOs and blockchain UI providers more generally to integrate with a broader grants ecosystem.
+In short, the motivation is greater social scalability for grants. We believe that a grant standard is necessary to coordinate grant efforts across the Ethereum community. A specified interface will enable wallets, DAOs and other blockchain UI providers to integrate with a broader grants ecosystem.
 
-TODO:
-* what kinds of solutions are out there already?
-* why do we need a standard? (e.g. what's the problem with the current status quo?)
-* how does having a standard for grants makes the situation better?
+The current process for grants in the Ethereum ecosystem is for organizations like the Ethereum Foundation or MolochDAO to internally debate and fund grants. This creates friction for the broader Ethereum community to participate in such efforts. Open grant proposals would allow for anyone in the Ethereum community to participate in the funding of necessary infrastructure through voting or awarding funds.
```

Comment:
What do you mean by "the current process"? I think it'd be useful to actually link examples of smart contracts or docs describing what MolockDAO, EF and others do to allocate grants right now.

When you say "Open grant proposals would allow for anyone in the Ethereum community to participate in the funding of necessary infrastructure through voting or awarding funds.", I'd also stress the added transparency/auditability of grants over time.